internal/node/otel.go

```diff
@@ -101,6 +101,14 @@ func (cli *OTELClient) Close(ctx context.Context) error {
 // ------------------------------------------------------------
 
 type OTELConfig struct {
+	// Resource contains information about the thing sourcing logs, metrics, and
+	// traces in OTEL. This information will be available in backends on all logs
+	// traces, and metrics that are generated from this source.
```

Comment:
Can you please add some details about the expected constraints?  eg: the resource is the service that initializes clues, it should represent the whole service and not just a part of it, etc.